packages/suite/src/utils/suite/analytics.ts

```diff
@@ -13,9 +13,31 @@ import {
     getPlatformLanguages,
 } from '@suite-utils/env';
 import { getCustomBackends } from '@suite-utils/backend';
+import { AccountTransactionBaseAnchor } from '@suite-constants/anchors';
 
+import type { AnalyticsState } from '@suite-reducers/analyticsReducer';
 import type { UpdateInfo } from '@trezor/suite-desktop-api';
 
+// redact transaction id from account transaction anchor
+export const redactTransactionIdFromAnchor = (anchor?: string) => {
+    if (!anchor) {
+        return undefined;
+    }
+
+    return anchor.startsWith(AccountTransactionBaseAnchor) ? AccountTransactionBaseAnchor : anchor;
+};
+
+export const hasUserAllowedTracking = (
+    enabled: AnalyticsState['enabled'],
+    confirmed: AnalyticsState['confirmed'],
+) => {
+    const isConfirmed = !!confirmed;
+    const isEnabled = isConfirmed ? !!enabled : true;
```

Comment:
this ternary condition is not wrong, just useless - value returned from ` hasUserAllowedTracking` will be the same without it:
```suggestion
    const isEnabled = !!enabled;
```